src/main/java/com/epam/lab/payments/UserValidator.java

```diff
@@ -28,19 +30,20 @@ public void validate(Object o, Errors errors) {
         UserDTO userDTO = (UserDTO) o;
 
         ValidationUtils.rejectIfEmptyOrWhitespace(errors, EMAIL, "NotEmpty");
-        if (userDTO.getEmail().length() < 6 || userDTO.getEmail().length() > 32) {
```

Comment:
Guys, it's very annoying, that i see equal changes in two different pull requests. U should read more about git flow, if u don't know how to fix it.